torchaudio/_extension/__init__.py

```diff
@@ -72,6 +73,15 @@
         _LG.debug("Failed to initialize ffmpeg bindings", exc_info=True)
 
 
+# Initialize rir-related features
+_IS_RIR_AVAILABLE = False
+if _IS_TORCHAUDIO_EXT_AVAILABLE:
+    _load_lib("libtorchaudio")
+
+    import torchaudio.lib._torchaudio  # noqa
+
```

Comment:
This part is now repeated multiple times, which is redundant. Can you merge the initialization logics around extension, kaldi, RIR as follow?

```
_IS_RIR_AVAILABLE = False
_IS_KALDI_AVAILABLE = False
_IS_TORCHAUDIO_EXT_AVAILABLE = is_module_available("torchaudio.lib._torchaudio")
if _IS_TORCHAUDIO_EXT_AVAILABLE:
    _load_lib("libtorchaudio")

    import torchaudio.lib._torchaudio # noqa

    _check_cuda_version()

    _IS_RIR_AVAILABLE = torchaudio.lib._torchaudio.is_rir_available()
    _IS_KALDI_AVAILABLE = torchaudio.lib._torchaudio.is_kaldi_available()
```